src/components/bcros/filing/addStaffFiling/modal/Form.vue

```diff
@@ -321,7 +322,28 @@ const handleSubmit = () => {
             </div>
             <div class="flex">
```

Comment:
As a result of https://github.com/bcgov/business-dashboard-ui/pull/144, old lines 314-346 no longer exist in main branch.

I think this feature branch should be rebased.

Reply:
Ok...